sdk/parents/azure-client-sdk-parent/pom.xml

```diff
@@ -1142,6 +1142,18 @@
                 <!-- Azure-core needs to open the test entities to Jackson for unit tests to operate -->
                 --add-opens com.azure.core/com.azure.core.implementation.entities=com.fasterxml.jackson.databind
                 --add-opens com.azure.core/com.azure.core.implementation.entities=ALL-UNNAMED
+
```

Comment:
These should not be in azure-client-sdk-parent. They are inherited by every child module. You should move these to the area path's SDK.
/cc @alzimmermsft

Reply:
Inherited from the child's pom.

Reply:
And there're some more `--add-opens` and `--add-exports` in the parent's pom, should they be moved to the children's poms too?

Reply:
@conniey - can you please provide an update on the above discussion ?
We are trying to close this for our next release - which is early next week, and would like to get this change in by Friday (code complete)